addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/type/generator/CcuMetadataExtractor.java

```diff
@@ -218,7 +218,7 @@ public UrlLoader(String url, String startLine, String endLine) throws IOExceptio
                         includeLine = false;
                     }
                 }
-                if ((includeLine == null || (startLine != null && includeLine)) && StringUtils.isNotBlank(line)) {
```

Comment:
Agree but I have seen worse will solving all these things, maybe for another PR?